skel/handlers/index.go

```diff
@@ -105,8 +105,8 @@ func Setup(app *sonic.Runtime) {
 	// Run LicenseRenewal
 	go daemons.LicenseRenewal("finex", app, vaultService)
 
-	// Run FetchMarkets
-	go daemons.FetchMarkets(peatioClient, vaultService, opendaxConfig.Addr)
+	// Run FetchPlatformConfiguration
+	go daemons.FetchPlatformConfiguration(peatioClient, vaultService, opendaxConfig.Addr)
```

Comment:
Is it `FetchPlatformConfiguration` or `FetchConfiguration`?

Reply:
FetchPlatformConfiguration is goroutine
FetchPlatform method where we create all needed configurations